csharp/src/Google.Protobuf/Collections/RepeatedField.cs

```diff
@@ -58,6 +58,27 @@ public sealed class RepeatedField<T> : IList<T>, IList, IDeepCloneable<RepeatedF
         /// <summary>
         /// Creates a deep clone of this repeated field.
         /// </summary>
+        public RepeatedField() : this(0)
+        {
+            
+        }
+
+        /// <summary>
+        /// Creates a deep clone of this repeated field.
+        /// </summary>
+        /// <param name="capacity">Capacity. The initial capacity of the internal buffer</param>
+        public RepeatedField(int capacity)
```

Comment:
What code would you expect to call this? User code doesn't generally create instances of RepeatedField; the fields are automatically initialized in messages.